corehq/apps/case_importer/tracking/models.py

```diff
@@ -15,6 +15,9 @@ class CaseUploadRecord(models.Model):
 
     upload_file_meta = models.ForeignKey('CaseUploadFileMeta', null=True)
 
+    class Meta(object):
+        index_together = ('domain', 'created')
+
```

Comment:
I imagine that _right_ now, it's no speed-up at all, since scanning a table with 50 rows in it is going to be as fast as filtering by an index first. Once there are a lot of rows in here, having this will make a difference. It's premature optimization, but of the kind that's so foreseeable as to probably count as just good practice.